src/components/DeleteModal.jsx

```diff
@@ -52,45 +51,45 @@ const DeleteModal = (props) => {
+                        {/* <ExclamationIcon
                           className="h-6 w-6 text-red-600"
                           aria-hidden="true"
-                        />
-                      </div>
+                        /> */}
+                      {/* </div> */}
                       <div className="mt-3 text-center sm:mt-0 sm:ml-4 sm:text-left">
                         <Dialog.Title
                           as="h3"
                           className="text-lg leading-6 font-medium text-gray-900"
                         >
-                          Delete Comment
+                          Delete comment
                         </Dialog.Title>
                         <div className="mt-2">
                           <p className="text-sm text-gray-500">
-                            Are you sure you want to delete this comment? This
-                            cannot be undone.
+                            Are you sure you want to delete this comment? This will remove the comment and
+                            can't be undone.
                           </p>
                         </div>
                       </div>
                     </div>
                   </div>
-                  <div className="bg-gray-50 px-4 py-3 sm:px-6 sm:flex sm:flex-row-reverse">
+                  <div className="px-4 py-3 sm:px-6 sm:flex sm:flex-row-reverse flex justify-center">
                     <button
                       type="button"
-                      className="w-full inline-flex justify-center rounded-md border border-transparent shadow-sm px-4 py-2 bg-red-600 text-base font-medium text-white hover:bg-red-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500 sm:ml-3 sm:w-auto sm:text-sm"
+                      className="uppercase w-full inline-flex justify-center rounded-md border border-transparent shadow-sm px-4 py-2 bg-[#ED6368] text-base font-medium text-white hover:bg-[#FFB8BB] focus:outline-none sm:ml-3 sm:w-auto sm:text-sm"
                       onClick={handleDelete}
                       id="confirm-delete-button"
                     >
-                      Delete
+                      Yes, Delete
                     </button>
                     <button
                       type="button"
-                      className="mt-3 w-full inline-flex justify-center rounded-md border border-gray-300 shadow-sm px-4 py-2 bg-white text-base font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 sm:mt-0 sm:ml-3 sm:w-auto sm:text-sm"
+                      className="uppercase mt-3 inline-flex justify-center rounded-md border border-transparent shadow-sm px-4 py-2 bg-[#67727E] text-base font-medium text-white hover:bg-[#E9EBF0] focus:outline-none sm:mt-0 sm:ml-3 sm:w-auto sm:text-sm"
```

Comment:
have a look at this link, can you maybe create custom colours instead of using hex colours in the code? 

https://tailwindcss.com/docs/adding-custom-styles